dataactbroker/permissions.py

```diff
@@ -20,6 +20,10 @@ def decorated_function(*args, **kwargs):
                     for permission in permissionList :
                         if(not userDb.hasPermission(user, permission)) :
                             validUser = False
+                        else:
```

Comment:
@bsweger Yes you are correct. The user should have at least one permission in the list to utilize the route.